spec/features/admin/admin_can_edit_user_profile_spec.rb

```diff
@@ -95,5 +95,29 @@
       expect(not_updated_user.name).to_not eq("April Dagonese")
       expect(not_updated_user.email).to_not eq("taken_email")
     end
+
+    it 'only updates user\'s slug when changing their email' do
```

Comment:
excellent thinking here! should probably do the same when changing an item's name etc too.